code/modules/surgery/bones.dm

```diff
@@ -22,6 +22,7 @@
 	return (affected.open >= 2 || (target.species.anatomy_flags & NO_SKIN)) && affected.stage == 0 && affected.status & ORGAN_BROKEN
 
 /datum/surgery_step/glue_bone/begin_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	tool.playsurgerysound(target, 75)
```

Comment:
All implementations of `begin_step` seem to call the base proc (`..()`), so instead of calling `playsurgerysound()` inside each implementation, you can do it from the base `/datum/surgery_step/proc/begin_step`.

Reply:
playsurgerysound() is not called by every implementation; if you think it'd be better, i can try and make an if statement in the base begin_step that does playsurgerysound() on certain steps but not others, but i feel like it'd be extremely clunky.

Reply:
Looks like every implementation *should* call it, though.
I suggest you use `SHOULD_CALL_PARENT(TRUE)` at the beginning of `/proc/begin_step` and add the supercall to the implementations that don't already have it.